src/app/items/containers/item-children-edit-list/item-children-edit-list.component.ts

```diff
@@ -117,17 +127,57 @@ export class ItemChildrenEditListComponent implements OnChanges {
     this.op?.toggle(event);
   }
 
+  updateChildPropagations(propagations: Partial<ItemPermPropagations>, childIdx: number): void {
+    this.childrenChanges.emit(
+      this.data.map((c, index) => {
+        if (index === childIdx) {
+          return {
+            ...c,
+            ...propagations,
+          };
+        }
+        return c;
+      })
+    );
+  }
+
   onContentViewPropagationChanged(contentViewPropagation: 'none' | 'as_info' | 'as_content'): void {
     this.op?.hide();
-    this.childrenChanges.emit(this.data.map((c, index) => {
-      if (index === this.propagationEditItemIdx) {
-        return {
-          ...c,
-          contentViewPropagation,
-        };
-      }
-      return c;
-    }));
+    if (!this.propagationEditItemIdx) throw new Error('Unexpected: Missed propagationEditItemIdx');
+    this.updateChildPropagations({ contentViewPropagation }, this.propagationEditItemIdx);
     this.propagationEditItemIdx = undefined;
   }
+
+  openAdvancedConfigurationDialog(child: PossiblyInvisibleChildData, childIdx: number): void {
```

Comment:
The name "openAdvancedConfigurationDialog" in the context of a "item children edit list" is not clear I think. It should be mention it is related to propagation. So for instance "openAdvancedPermPropagationConfigurationDialog"